### ISSUE TITLE
Android error

### ISSUE THREAD

**Issue by agent3bood** [Netvent/storyly-mobile#19]
I have a minimal example
```
<Storyly
          style={{width: '100%', height: 120}}
          storylyId="xdxdxd"
        />
```

The app will show error message once the screen has loaded.
```
Error while updating property 'storylyInit' of a view managed by: STStoryly

null

storylySegment
```
```
"react-native": "0.61.5",
"storyly-react-native": "^1.3.0",
```

**Comment by agent3bood** [comment]
To fix this crash I added `storylySegments` prop, please add this to the documentations and maybe set a default props. 

Thanks

**Comment by levent-oral** [comment]
Hi @agent3bood ,

This issue has been fixed with storyly-react-native@1.3.1 version. 

Thanks!

**Comment by agent3bood** [comment]
Thanks.
